doc/api/deprecations.md

```diff
@@ -2519,18 +2519,22 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`][] and
 [`request.socket`][] over [`request.connection`][].
 
-<a id="DEP0134"></a>
-### DEP0134: process._tickCallback
+<a id="DEP0XXX"></a>
```

Comment:
Why is this changing? Same comment on the next line.

Reply:
apparently, I forgot how to rebase here :smile: